packages/ocaml-variants/ocaml-variants.4.07.1+musl+static+flambda/opam

```diff
@@ -2,7 +2,7 @@ opam-version: "2.0"
 synopsis:
   "Official 4.07.1 release compiled with musl-gcc -static and with flambda activated"
 description:
-  "Requires musl-gcc to be installed (package musl on Arch Linux or musl-tools on Debian)"
+  "Requires musl-gcc to be installed (package musl on Arch Linux, musl-tools on Debian, musl-dev on Alpine)"
 maintainer: "platform@lists.ocaml.org"
 depends: [
   "ocaml" {= "4.07.1" & post}
```

Comment:
Below (and in various other opam files) I see that `musl-gcc` is called directly. I think you need to modify your sed script to consider also that case